extensions/ql-vscode/src/view/result-tables.tsx

```diff
@@ -85,12 +83,16 @@ export class ResultTables
         // out.
         schema: { name: ALERTS_TABLE_NAME, version: 0, columns: [], tupleCount: 1 },
         name: ALERTS_TABLE_NAME,
-        ...this.props.interpretation,
+        ...props.interpretation,
       });
     }
     return resultSets;
   }
 
+  private getResultSets(): ResultSet[] {
```

Comment:
Looks like this isn't used.

Wait...it is. Though, not sure what the difference is between this function and the static `_getResultSetsOfProps`.

Reply:
I just factored out the body of that method into a static method that takes a props argument, so that I could call it from `getDerivedStateFromProps`.